Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM gcc:latest as build
+
+WORKDIR /app
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y cmake
+
+COPY . /app/
+
+RUN sed -i 's/MARIADB_CONNECTOR_COMPILE_JOBS__=1/MARIADB_CONNECTOR_COMPILE_JOBS__=2/' /app/CMakeVariables.txt 
```

Comment:
This should be looking for `MARIADB_CONNECTOR_COMPILE_JOBS__=` instead of `MARIADB_CONNECTOR_COMPILE_JOBS__=1` as the latter will not be found if we on the master repo change it from the default

Reply:
will need to modify the sed to just replace the whole line then